src/archivematicaCommon/lib/elasticSearchFunctions.py

```diff
@@ -29,22 +29,20 @@
 import re
 import sys
 import time
-from lxml import etree
 
 from django.db.models import Min, Q
-from main.models import File, Identifier, Transfer
+from elasticsearch import Elasticsearch, ImproperlyConfigured
+from elasticsearch.helpers import bulk
+from lxml import etree
+from six.moves import range
```

Comment:
I did! My code editor was actually auto-applying isort due to a plugin - I've since fixed that in the plugin config, but figured might as well keep the changes since I was working under the assumption that the related ADR was approved but just not implemented yet.

Reply:
It took me a while to make the logic leap, but that's cool.

Reply:
I think my comment was ambiguous! I *was* working under that assumption, but given that the ADR isn't approved, I'll return this to how it was for now. Thanks!